applications/luci-app-commands/luasrc/view/commands.htm

```diff
@@ -109,8 +109,7 @@
 		if (legend && output)
 		{
 			var prefix = location.protocol + '//' + location.hostname +
-			           (location.port ? ':' + location.port : '') +
-					   location.pathname.split(';')[0] + 'command/';
+			           (location.port ? ':' + location.port : '') + '<%=url('command')%>/';
```

Comment:
Sorry for the delay.  Please also sign off the commit as required by the CONTRIBUTING.md guide.

While at it, can you also tuck the ending slash `/` inside the `url()` call, like `url('command/')`?  Thank you